.circleci/config.yml

```diff
@@ -1,22 +1,24 @@
 version: 2.1
 
-jobs:
-  test:
-    docker:
-      - image: circleci/php:7.3.3-stretch-node-browsers
-    steps:
-      - checkout
-      - prepare-environment
-      - run: composer phpcs
-
-commands:
- prepare-environment:
-   description: "Install dependencies."
-   steps:
-     - run: composer install
+orbs:
+  php: circleci/php@1.0
+  wp-svn: studiopress/wp-svn@0.1
 
 workflows:
-  version: 2
-  check-wp-cs:
+  test-deploy:
     jobs:
-      - test
+      - php/test:
```

Comment:
This basically does the same thing as the previous `test` job, but it handles all of the setup on its own.

Like getting the right docker image and doing `composer install` with caching.